plugin panic

plugin panic a lot and throws error: 

```
plugins-storage/sources/gop-968684667/src/github.com/containeroo/duplicateheader/duplicateheader.go:30:13: panic
```

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ experimental:
   plugins:
     duplicateheader:
       moduleName: "github.com/containeroo/duplicateheader"
-      version: "v1.0.0"
+      version: "v1.0.1"
 ```
 
 ```yaml

--- a/duplicateheader.go
+++ b/duplicateheader.go
@@ -27,10 +27,11 @@ func New(ctx context.Context, next http.Handler, config *Config, name string) (h
 	}
 
 	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
-		source := req.Header.Get(config.Source)
-		if source != "" {
-			for _, dest := range config.Destination {
-				req.Header.Set(dest, source)
+		if source, ok := req.Header[config.Source]; ok {
+			if source[0] != "" {
+				for _, dest := range config.Destination {
+					req.Header.Set(dest, source[0])
+				}
 			}
 		}
 		next.ServeHTTP(rw, req)

